frontend/src/components/Footer.tsx

```
@@ -0,0 +1,82 @@
+                {/* Brand Section */}
+                <div className="md:col-span-2 space-y-6">
+                    <div className="inline-block relative">
+                        <h2 className="text-6xl font-zilla font-bold text-[#FFF4EA] relative z-10">
+                            papertrail.
+                        </h2>
+                        {/* Decorative Underline/Shadow */}
+                        <div className="absolute -bottom-2 left-0 w-full h-4 bg-[#EDDCC6] -z-0 transform -rotate-1 opacity-50"></div>
+                    </div>
+
+                    <p className="text-[#FFF4EA] font-mono text-sm max-w-md leading-relaxed border-l-2 border-[#EDDCC6] pl-4">
+                        // OPEN SOURCE . SELF HOSTED<br />
```

Biome (2.3.14)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)